### PR TITLE
[SDK-3828] update ionic react guide for react v2 ga

### PR DESCRIPTION
**Pull request by ewanharris**
Updates the ionic + react quickstart for the upcoming v2 of react

### CODE REVIEW

File: articles/quickstart/native/ionic-react/01-login.md [hunk 1]
```diff
@@ -158,14 +164,18 @@ import { IonButton } from '@ionic/react';
...
-    logout({ localOnly: true });
+    await logout({
+      async openUrl(url) {
+         // Redirect using Capacitor's Browser plugin
+        await Browser.open({
+          url,
+          windowName: "_self"
+        });
+      }
+    });
```
Comment (frederikprijck) [inline articles/quickstart/native/ionic-react/01-login.md]:
```suggestion
    await logout({
      logoutParams: {
        returnTo: logoutUri,
      },
      async openUrl(url) {
         // Redirect using Capacitor's Browser plugin
        await Browser.open({
          url,
          windowName: "_self"
        });
      }
    });
```

File: articles/quickstart/native/ionic-react/files/logout-button.md [hunk 2]
```diff
@@ -13,14 +13,18 @@ import { IonButton } from '@ionic/react';
...
-    logout({ localOnly: true });
+    await logout({
+      async openUrl(url) {
+         // Redirect using Capacitor's Browser plugin
+        await Browser.open({
+          url,
+          windowName: "_self"
+        });
+      }
+    });
```
Comment (frederikprijck) [inline articles/quickstart/native/ionic-react/files/logout-button.md:30]:
```suggestion
    await logout({
      logoutParams: {
        returnTo: logoutUri
      },
      async openUrl(url) {
         // Redirect using Capacitor's Browser plugin
        await Browser.open({
          url,
          windowName: "_self"
        });
      }
    });
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — DISMISSED** [review]

File: articles/quickstart/native/ionic-react/interactive.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ Still having issues? Check out our [documentation](https://auth0.com/docs) or vi
 
 ::::checkpoint
 ::: checkpoint-default
-Use the `buildAuthorizeUrl` function to construct the login URL, and the `Browser.open` function to open it using the platform's system browser component for a way your users log in. Users redirect to the login page at Auth0 and do not receive any errors.
+The `loginWithRedirect` function tells the SDK to initiaze the login flow, using the `Browser.open` function to open the login URL with the platform's system browser component by setting the `openUrl` parameter. This provides a way for your user to log in to your application. Users redirect to the login page at Auth0 and do not receive any errors.
```
Comment (ewanharris) [inline articles/quickstart/native/ionic-react/interactive.md]:
```suggestion
The `loginWithRedirect` function tells the SDK to initiate the login flow, using the `Browser.open` function to open the login URL with the platform's system browser component by setting the `openUrl` parameter. This provides a way for your user to log in to your application. Users redirect to the login page at Auth0 and do not receive any errors.
```

File: articles/quickstart/native/ionic-react/files/logout-button.md [hunk 4]
```diff
@@ -13,14 +13,29 @@ import { IonButton } from '@ionic/react';
...
+        returnTo: logoutUri
+      },
+      async openUrl(url) {
+         // Redirect using Capacitor's Browser plugin
+        await Browser.open({
+          url,
+          windowName: "_self"
+        });
+      }
+    });
```
Comment (frederikprijck) [inline articles/quickstart/native/ionic-react/files/logout-button.md:30]: GitHub suggestion went wrong apparantly.

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**